### ISSUE TITLE
submitting PR fails

### ISSUE THREAD

**Issue by gamerson** [node-gh/gh#721]
**Version number**
In which version does the bug appear?
2.5.0

**Describe the bug**

I've got a local branch and I'm submitting a pull request to another user on the repo with this command:

```
~/my repos/liferay-blade-cli(BLADE-483 ✔) gh pr -s christopherbrianboyd -t "BLADE-483"
Submitting pull request to @christopherbrianboyd
Total 0 (delta 0), reused 0 (delta 0)
remote:
remote: Create a pull request for 'BLADE-483' on GitHub by visiting:
remote:      https://github.com/gamerson/liferay-blade-cli/pull/new/BLADE-483
remote:
To github.com:gamerson/liferay-blade-cli.git
 * [new branch]        BLADE-483 -> BLADE-483
Error: Can't submit pull request
Error: Error submitting PR
HttpError: Not Found
    at _submitHandler (/usr/local/lib/node_modules/gh/lib/cmds/pull-request.js:835:15)
    at processTicksAndRejections (internal/process/next_tick.js:81:5)
```

Any ideas?

**Comment by protoEvangelion** [comment]
@gamerson can you try v 2.5.1? `npm i gh@2.5.1`

I think this could be related to https://github.com/node-gh/gh/issues/717

**Comment by gamerson** [comment]
Its fixed in 2.5.1.  Thanks for super fast turn-around.

**Comment by protoEvangelion** [comment]
Thanks for the bug report! I can't catch em all 🏈
